exercises/collatz-conjecture/canonical-data.json

```diff
@@ -0,0 +1,36 @@
+{
+  "exercise": "collatz-conjecture",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "zero steps for one",
+      "property": "collatz",
+      "number": 1,
+      "expected": 0
+    },
+    {
+      "description": "divide if even",
+      "property": "collatz",
+      "number": 16,
+      "expected": 4
+    },
+    {
+      "description": "even and odd steps",
+      "property": "collatz",
+      "number": 12,
+      "expected": 9
+    },
+    {
+      "description": "zero is an error",
+      "property": "collatz",
+      "number": 0,
+      "expected": -1
```

Comment:
Would you please update the expected error value:

From: https://github.com/exercism/x-common#test-data-format-canonical-datajson
> if an error is expected (because the input is invalid, or any other reason), the value at "expected" should be an object containing exactly one property, "error", whose value is a string.
>     The string should explain why the error would occur.